src/Commands/Site/InfoCommand.php

```diff
@@ -36,6 +37,7 @@ class InfoCommand extends SiteCommand
      *     owner: Owner
      *     frozen: Is Frozen?
      *     last_frozen_at: Date Last Frozen
+     * @default-fields id,name,label,created,framework,region_label,organization,plan_name,max_num_cdes,upstream,holder_type,holder_id,owner,frozen,last_frozen_at
```

Comment:
You should use `@default-table-fields` here. If you use `@default-fields`, then you will also filter out unlisted fields from yaml and json formats et. al., and `region` should definitely be in the machine formats by default.

Optionally, you could use `@default-fields` to exclude `region_label` from the machine formats by default, and `@default-table-fields` to exclude `region` from the human-readable formats by default.